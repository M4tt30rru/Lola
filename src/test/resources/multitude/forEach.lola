##Import "std.lola"
##Find for (##Any(loop) | ##Expression(filter)) { ##Any(statements) }
 ##replace for (##(loop)) {if (!(##(filter))) continue; ##(statements)}

for (final Integer i : nums() | i % 2 == 0) {
	f(i);
}
