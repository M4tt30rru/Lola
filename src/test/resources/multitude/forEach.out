for (final Integer i : nums()) {
 if (!(i%2==0))
  continue;
 f(i);
}
